data/org.freedesktop.portal.Device.xml

```diff
@@ -58,6 +58,66 @@
       <arg type="a{sv}" name="options" direction="in"/>
       <arg type="o" name="handle" direction="out"/>
     </method>
+
+    <!--
+        OpenPipeWireRemote:
+        @options: Vardict with optional further information
+        @fd: File descriptor of an open PipeWire remote.
+
+        Open a file descriptor to the PipeWire remote where the microphone,
+        speakers, and camera nodes are available. The file descriptor should be
+        used to create a <classname>pw_core</classname> object, by using
+        <function>pw_context_connect_fd</function>.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>devices as</term>
+            <listitem><para>
+              The devices to access. Supported devices for this call are 'camera',
+              'microphone', and 'speakers'.
+            </para></listitem>
```

Comment:
Note: the `speakers` permission should control the application's ability to capture output sound from the output device. The application shouldn't need any special permissions to play sound, so as to be consistent with other platforms' permission model.

Reply:
Mechanically, this is not true: applications in a Flatpak sandbox have no way to play sound unless their permissions allow for it, either statically (`--socket=pulseaudio`, `--socket=pipewire` or maybe `--device=all`), or dynamically (something like this interface).

I agree it would make sense for this interface to have a way to grant audio output permission to apps statically, rather than prompting the user - but that would have to be implemented by the app asking xdg-desktop-portal for access to the speakers, and xdg-desktop-portal replying "yes, here is your file descriptor" without asking the user's permission first. This is analogous to the way the OpenUri portal will open `http:` and `https:` URLs in a web browser without asking permission (again, to match app-author expectations from other platforms), but the app still has to ask the portal to do it.

If capturing the mixed output of all applications (a "monitor" in PulseAudio terms, I don't know what Pipewire calls this) is something that sandboxed apps will want to do, then I think that would be a fourth device in this interface.